webapp/plugins/vue-filters.js

```diff
@@ -39,15 +38,6 @@ export default ({ app = {} }) => {
       }
       return accounting.formatNumber(value || 0, precision, thousands, decimals)
     },
-    // format seconds or milliseconds to durations HH:mm:ss
-    duration: (value, unit = 's') => {
-      if (unit === 'ms') {
-        value = value / 1000
-      }
-      return value
-        ? format(addSeconds(new Date('2000-01-01 00:00'), value), 'HH:mm:ss')
-        : '00:00:00'
-    },
```

Comment:
nice that you figured out we don't use this, removing unused dependencies should be an ongoing task